.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: Java CI
+
+on: [ push ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest , windows-latest , macos-latest ]
+        java: [ 11 ]
+        experimental: [ false ]
+
+    steps:
+      - uses: actions/checkout@v2.3.2
+      - uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Set up JDK ${{ matrix.java }}
+        uses: actions/setup-java@v1.4.0
+        with:
+          java-version: ${{ matrix.java }}
+      - name: Build with mvnw
+        run: |
+          chmod 777 ./mvnw
```

Comment:
This should not be necessary. `mvnw` is (or should be) committed with the proper executable permissions already